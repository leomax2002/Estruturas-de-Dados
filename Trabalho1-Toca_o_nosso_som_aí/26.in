add SambaMakossa
add Televisao
add Stab
add MonstroInvisivel
del SambaMakossa
del Televisao
del Stab
del MonstroInvisivel
list
undo
list
undo
list
undo
list
undo
list
play
del SambaMakossa
current
ended
del Televisao
current
ended
del Stab
current
ended
del MonstroInvisivel
current
ended
del SambaMakossa
current
fight