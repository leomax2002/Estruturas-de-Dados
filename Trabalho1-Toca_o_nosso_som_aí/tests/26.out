[vazia]
MonstroInvisivel
Stab,MonstroInvisivel
Televisao,Stab,MonstroInvisivel
SambaMakossa,Televisao,Stab,MonstroInvisivel
SambaMakossa
Televisao
Stab
MonstroInvisivel
SambaMakossa
Jedi Wagner, assuma o comando!